scripts/wasm/upload-contracts.sh

```diff
@@ -15,7 +15,7 @@ fi
 TXFLAG="--gas-prices 0.01$DENOM --gas auto --gas-adjustment 1.3 -y -b block --chain-id $CHAIN_ID --node $NODE --output json"
 
 REPO=https://github.com/public-awesome/contracts
```

Comment:
Have two repos now and the new open source contracts will live at https://github.com/public-awesome/stargaze-contracts. Working on getting CI working w/ publishing the artifacts. Also no longer need the Github token for these.

Marketplace contracts are now at https://github.com/public-awesome/marketplace. Also need to setup CI, but this will need the Github token.

Reply:
New stargaze-contracts release: https://github.com/public-awesome/stargaze-contracts/releases